pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -526,7 +525,7 @@ func handleLastRebootReason(ctx *nodeagentContext) {
 	// until after truncation.
 	var rebootStack = ""
 	ctx.rebootReason, ctx.rebootTime, rebootStack =
-		agentlog.GetCurrentRebootReason()
+		agentlog.GetCommonRebootReason()
 	if ctx.rebootReason != "" {
 		log.Warnf("Current partition RebootReason: %s\n",
```

Comment:
Also delete the block below where it calls GetCommonRebootReason, and add a comment before GetOtherRebootReason that this is to handle old versions in the other partition.

Reply:
Done.